### DISCUSSION TITLE
命令行工具是否能增加菜单与权限导出？

### DISCUSSION THREAD

**Discussion by YouLiLiao** [slowlyo/owl-admin#110] [FAQ]
因为在本地开发时配置好的数据，部署时需要迁移。从dcat-admin转过来的，还有点不习惯

**Comment by slowlyo** [comment]
https://github.com/owl-admin/system-backup

看看这个包

**Reply by YouLiLiao** [reply]
OK，谢谢！

**Reply by YouLiLiao** [reply]
添加备份后，在另一台电脑恢复的时候没有作用

**Reply by YouLiLiao** [reply]
没事了，是备份加密了，然后两边的APP_KEY不一样导致的

**Reply by YouLiLiao** [reply]
![image](https://github.com/slowlyo/owl-admin/assets/11286354/e21345bd-2f6b-4d74-bec5-5c8943b27d16)
![image](https://github.com/slowlyo/owl-admin/assets/11286354/86ca394b-ab5d-49a5-95b8-ffa9e9007af8)
![image](https://github.com/slowlyo/owl-admin/assets/11286354/837f52ca-974b-4ce2-8200-a88856bef65f)
又发现新问题了，备份的时候，管理员表把roles关联也备份进去了，然后在恢复的时候，由于AdminUser的guarded属性设置空数组允许所有属性插入数据库，然后导致了报错

**Reply by slowlyo** [reply]
可以提个pr

**Reply by YouLiLiao** [reply]
已提交PR

**Reply by slowlyo** [reply]
发版了
